docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -13,18 +28,39 @@ default::
 [source,yaml]
 ----
 channel: stable-4.8
-clusterID: ${openshift:clusterID}
 upstream: https://api.openshift.com/api/upgrades_info/v1/graph
 ----
 
-See the https://docs.openshift.com/container-platform/latest/updating/updating-cluster-between-minor.html[OpenShift docs] for available paramters and values.
+See the https://docs.openshift.com/container-platform/latest/updating/updating-cluster-between-minor.html[OpenShift docs] for available parameters and values.
+
+[NOTE]
+====
+Field `clusterID` must be set in the `ClusterVersion` object.
+The value for this field is supposed to be extracted from the cluster as a fact.
+However, the corresponding dynamic fact isn't implemented yet.
```

Comment:
Is there an issue to have this addressed?

Reply:
I created an issue now [OCP-491 (internal)](https://ticket.vshn.net/browse/OCP-491)